cmd/openshift-router/cmd.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"math/rand"
+	"os"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	"k8s.io/kubernetes/pkg/util/logs"
+
+	irouter "github.com/openshift/origin/pkg/cmd/infra/router"
+	"github.com/openshift/origin/pkg/cmd/util/serviceability"
+
+	// install all APIs
+	_ "github.com/openshift/origin/pkg/api/install"
+	_ "k8s.io/kubernetes/pkg/api/install"
+	_ "k8s.io/kubernetes/pkg/apis/autoscaling/install"
+	_ "k8s.io/kubernetes/pkg/apis/batch/install"
+	_ "k8s.io/kubernetes/pkg/apis/extensions/install"
+)
+
+func main() {
+	logs.InitLogs()
```

Comment:
Can you grab my "standardization commit" and bring it in to these?  It turns all of this into generic, can't mess up code.  If we're going to have tons of these binaries, the standardization one is important.

https://github.com/openshift/origin/pull/15106

Note this will make the openshift router image about 20M larger and make our builds 25% larger.  Going from 40M zipped to 50M zipped isn't the end of the world, but these are going to start adding up.